### ISSUE TITLE
FISH-14 - boilerplate > use `@nx/node:application` generator to bootstrap `cli` app

### ISSUE THREAD

**Issue by wijohnst** [christopher-spencer/something-fishy#14]
## TL/DR

Use the `@nx/node:application` generator to add a new node application called `cli` to the `/app` directory. 

## Description

We are going to build out a node app called `cli` that will handle different command-line tasks for our monorepo. We can use the [@nx/node:application](https://nx.dev/nx-api/node/generators/application) plugin to add the boilerplate code for a node application. 

## Acceptance Criteria

- [x] - `/app` directory is added
- [x] - `cli` typescript / node app is added to `/app`
- [x] - other associated linting (`eslint`) , testing (`jest`), and formatting (`prettier`) files are added to root (this will happen automatically when you run the generator)
- [ ] - `app/cli/tsconfig.app.json` is updated to `"module": "NodeNext"`
